.devcontainer/devcontainer.json

```
@@ -19,16 +18,15 @@
                        "**/.markdownlint-cli2.yaml": true,
                        "**/.pre-commit-config.yaml": true,
                        "**/Dockerfile": true,
-                       "**/mkdocs.yml": true,
-                       "**/scripts": true
+                       "**/mkdocs.yml": true
   },
                 "editor.tabSize": 8,
                 "editor.indentSize": 4,
                 "editor.detectIndentation": false,
                 "r.lsp.diagnostics": false,
                 "r.plot.useHttpgd": true,
                 "r.rpath.linux": "/usr/bin/R",
-                "r.rterm.linux": "/usr/bin/R",
+                "r.rterm.linux": "/workspaces/r-dev-env/scripts/launch_r.sh",
```

The root directory may not be `/workspaces/r-dev-env/`, so let's have this as 
```suggestion
                "r.rterm.linux": "/usr/bin/R",
```
to start with and use `localscript.sh` to update this to `"$WORK_DIR/scripts/launch_r.sh"` (it must be a full path, so we can't simply use `./scripts/launch_r.sh` here)